src/style.js

```diff
@@ -107,7 +107,7 @@ export function setImportantStyles(element, styles) {
 
 /**
  * Sets the CSS style of the specified element with optional units, e.g. "px".
- * @param {Element} element
+ * @param {?Element} element
```

Comment:
`?` not necessary since classes are nullable by default.

Reply:
I thought the style was to always be explicit?

Reply:
yes, Google style guide states that this should be explicit

https://google.github.io/styleguide/jsguide.html

"...all types except primitives and record literals must be annotated explicitly with either ? or ! to indicate whether they are nullable or not...."

Reply:
Fair, though you'll notice we do not strictly adhere to the Google style guide. :) No action required.